content/en/docs/security/otel-collector-hosting-best-practices.md

```diff
@@ -0,0 +1,56 @@
+---
+title: OpenTelemetry Collector Hosting Best Practices
+description: Follow best practices to securely host OpenTelemetry Collector.
+weight: 10
+---
+
+When setting up hosting for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your hosting instance.
+
+## Storing Configuration Information Securely
+
+<!--- TODO: SHOULD ensure sensitive configuration information is stored securely. How? -->
+
+## Permissions
+
+<!--- TODO: SHOULD not run the OpenTelemetry Collector as root/admin user. Why? (Give the reader motivation.) How do you do that?
+- NOTE: MAY require privileged access for some components -->
```

Comment:
The Collector SHOULD NOT require privileged access, except where the data it's obtaining is in a privileged location. For instance, in order to get pod logs by mounting a node volume, the Collector daemonset needs enough privileges to get that data.

The rule of least privilege applies here.